src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -58,7 +58,10 @@ std::vector<std::string> GetModuleReadOnlyParams(
     module_ro_params.push_back(kSignalChangeThreshold);
     module_ro_params.push_back(kState);
     module_ro_params.push_back(kSisData);
+  } else if (enums_value::kLight == module_type) {
+    module_ro_params.push_back(kStatus);
```

Comment:
kStatus is a bit generic and non-descriptive. Maybe rename to kLightStatus?

Reply:
@JackLivio fixed at d3528aa4f52db0c2fbac0e1f134b8fbef8f5bd6b , please take a look